README.md

```diff
@@ -45,7 +45,7 @@ list of protocol features](https://docs.rs/rustls/latest/rustls/manual/_04_featu
 
 While Rustls itself is platform independent, by default it uses [`aws-lc-rs`] for implementing
 the cryptography in TLS.  See [the aws-lc-rs FAQ][aws-lc-rs-platforms-faq] for more details of the
-platform/architecture support constraints in aws-lc-rs.
+platform/architecture support constraints in [`aws-lc-rs`].
```

Comment:
I don't think we commonly spell crate names as literals (at least I personally normally don't use code literals for crate names, except when used in the context of actual code).

Reply:
Yeah, agree. Probably some instants of not following this rule elsewhere in the crate.